NDB/clusterj-fixes/clusterj-fix.patch

```diff
@@ -15,21 +16,22 @@ index 6f2abce3..c03898ca 100644
 -            super.finalize();
 -        }
 -    }
-+//   protected void finalize() throws Throwable {
-+//       try {
-+//           if (delegate != null) {
-+//               delegate.release();
-+//           }
-+//       } finally {
-+//           super.finalize();
-+//       }
-+//   }
++//    @SuppressWarnings("deprecation")
```

Comment:
Why do we comment out this code?

Reply:
As discussed earlier the code in file "NDB/clusterj-fixes/clusterj-fix.patch" is reorganized but it is exactly the same as before.  We have removed the functionality of relying on finalize methods for clean-up because under load they become bottle neck in Java GC.